veggieseasons/.gitignore

```diff
@@ -1,3 +1,6 @@
+# App is designed specifically for iOS to showcase Cupertino package.
```

Comment:
nit: A little extra detail here might help:

"This app is designed to show how to use Flutter's cupertino package to build UI for iOS. It's not intended to run on Android, desktop, or web."